src/webportal/src/app/cluster-view/services/services.component.js

```diff
@@ -16,14 +16,17 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 // module dependencies
-import { getServiceView } from './service-info';
+const { getServiceView } = require('./service-info');
 
 const breadcrumbComponent = require('../../job/breadcrumb/breadcrumb.component.ejs');
 const loadingComponent = require('../../job/loading/loading.component.ejs');
 const serviceTableComponent = require('./service-table.component.ejs');
 const serviceViewComponent = require('./services.component.ejs');
 const loading = require('../../job/loading/loading.component');
 const webportalConfig = require('../../config/webportal.config.js');
+require('bootstrap');
+require('admin-lte/dist/css/AdminLTE.min.css');
+require('bootstrap/dist/css/bootstrap.css');
```

Comment:
did you remove admin-lte and bootstrap in this PR?

Reply:
Some old pages still needs admin-lte's css styles. So the admin-lte library won't be removed until all old pages are refactored.

This pull request replaces the layout rendered by admin-lte with a new react/fabric based implementation. The admin-lte javascript dependencies are removed.